src/Codeception/Lib/GroupManager.php

```diff
@@ -118,6 +118,12 @@ public function groupsForTest(\PHPUnit\Framework\Test $test)
                 if (strpos($filename . ':' . $test->getName(false), $testPattern) === 0) {
                     $groups[] = $group;
                 }
+                // TODO: consider mb_strtolower per https://stackoverflow.com/a/5473569
```

Comment:
Is there a case where `metadata()` method does not exist ?

Reply:
In some tests it did not so I added the check to make sure it was there before I called it